www/src/components/shell/terminal/theme/Selector.tsx

```diff
@@ -0,0 +1,67 @@
+import { forwardRef, useContext } from 'react'
+import { Button, Flex } from 'honorable'
+import { ListBoxItem, Select, SprayIcon } from '@pluralsh/design-system'
+
+import { normalizedThemes, themeNames } from './themes'
+import { TerminalThemeContext } from './context'
+
+const ThemeSelectButton = forwardRef<any, any>((props, ref) => (
+  <Button
+    ref={ref}
+    secondary
+    small
+    width={32}
+    height={32}
+    {...props}
+  >
+    <SprayIcon alignSelf="center" />
+  </Button>
+))
```

Comment:
Shouldn't it be `IconFrame`?

Reply:
I have followed the `Select` stories to create a custom trigger button. IIRC there were some console errors when I tried to use different base component.